tests/utils/queryApiGateway.test.ts

```diff
@@ -31,7 +31,7 @@ const goodRequest = {
 
 // Test queryApiGateway()
 describe('Querying the API Gateway', () => {
-  const goodResponse = { ClaimType: 'PUA' }
```

Comment:
Note: This is using the old data structure, so I updated it to use current data structure.